impeller/renderer/backend/vulkan/allocator_vk.h

```diff
@@ -45,6 +45,8 @@ class AllocatorVK final : public Allocator {
       StorageMode mode,
       const TextureDescriptor& desc) override;
 
+  ISize GetMaxTextureSizeSupported() override;
```

Comment:
Here and elsewhere, the overridden functions need to have a comment that mentions the base class that contains the method being overriden. So `// |Allocator|` here like for the functions above.

Reply:
Done.